libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -325,7 +325,7 @@ bool ReferencesResolver::visit(InlineAssembly const& _inlineAssembly)
 		analysisInfo,
 		errorsIgnored,
 		errorTypeForLoose,
-		yul::EVMDialect::looseAssemblyForEVM(EVMVersion::constantinople()),
+		yul::EVMDialect::looseAssemblyForEVM(EVMVersion::petersburg()),
```

Comment:
Shouldn't this just use the default constructor instead?

Reply:
I think #5959 also touches on these so would prefer to merge that first.

Reply:
Ah I see, this just updates all references to `constantinople` by the more or less identical `petersburg`.